src/components/YouTubePlayer.tsx

```diff
@@ -105,11 +105,16 @@ const YouTubePlayer = ({
     }
 
     (player as any).seekTo(seconds, true);
+
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [player, seconds]);
 
   // broadcast current player time as it changes
   const [counter, setCounter] = React.useState(0);
+
+  //TODO : Need a better way to get the current time
+  (window as any).$gsecs = String(seconds);
```

Comment:
Could you explain the purpose of this line?

Reply:
Ah, I see that this is related to the "Set End Time" button. Very handy! I will have to study the code more but I suspect that we can use our state management functions to handle this rather than the global variable, however this seems to work for now.

Reply:
Believe me this cut down the time required to do stamping to half. Now mostly there is no need to set time manually.
The drop down tags have reduced the time, no need to type anything except title and description.
I added a new issue about the global var, so we don't forget.